### DISCUSSION TITLE
Test build for grid layout

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#230] [General]
Need some feedback for the grid layout. See #74 and #166
To toggle the grid you need to add the `Grid` action to your dock

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.8-test.1'>Test build for grid layout</a>.</em>

**Comment by ltguillaume** [comment]
- All individual shortcuts somehow have the full height of the available space for the result list, causing the icons to be all over the place.
- Scrolling is impossible (maybe because of the shortcuts' height issue): scrolling just jumps everywhere, and may even jump back to the top position as soon as you release your finger. 
- The reloading and the grid actions have animating icons

**Reply by TBog** [reply]
* I found the issue with the shortcuts. The layout xml for them took all the available vertical space. Will be fixed in the next release.
The layout works when used in a GridView but is messed up in the RecycleView
* Scrolling is messed up by having shortcuts with invalid layout on the first/bottom row.
* i just reused some drawables I already had. Besides the unwanted animation should I keep them?

**Reply by ltguillaume** [reply]
They don't really match the other icons, but I think it's fine. Is the launcher icon exactly the same as the grid view?

**Reply by TBog** [reply]
The launcher icon is using the same drawable.
The refresh is using the "loading" arrow icon

**Reply by ltguillaume** [reply]
Yeah I would say the reload/loading is quite different from the arrows in Most accessed/Recent and frequent/Adaptive history, for example, but grid looks fine.

**Comment by TBog** [comment]
Fixed some issues. If you have time please check this new test build.
[Test build 2 for grid layout](https://github.com/TBog/TBLauncher/releases/tag/v6.8-test.2)

**Reply by ltguillaume** [reply]
Grid layout looks good now.
